packages/compass-preferences-model/src/preferences.ts

```diff
@@ -579,51 +499,48 @@ const modelPreferencesProps: Required<{
       long: 'Force connection string properties to take specific values',
     },
     customPostProcess: parseRecord,
+    validator: Joi.array()
+      .items(Joi.array().items(Joi.string(), Joi.string()))
```

Comment:
I think if you want tuples, this is how you’d do it:

```suggestion
      .items(Joi.array().length(2).items(Joi.string()))
```